tests/utils.go

```diff
@@ -1694,8 +1695,47 @@ func removeNamespaces() {
 	fmt.Println("")
 	for _, namespace := range testNamespaces {
 		fmt.Printf("Waiting for namespace %s to be removed, this can take a while ...\n", namespace)
-		EventuallyWithOffset(1, func() bool { return errors.IsNotFound(virtCli.CoreV1().Namespaces().Delete(namespace, nil)) }, 240*time.Second, 1*time.Second).
-			Should(BeTrue())
+		EventuallyWithOffset(1, func() bool {
```

Comment:
I might misunderstood what's happen here, but I think it is better to split this function into some sub-functions returning errors and asserting on that.
As it is now, the test only fails when `IsNotFound` is false, but as I understand it, there are a lot more errors along the way that are just printed and could be missed. should they fail the test?

Reply:
Any error that is not `IsNotFound` should fail this is the current state
I also about to change this function to return `error` instead of `bool` as you requested